WordPress/Classes/Utility/Editor/EditorFactory.swift

```diff
@@ -26,6 +26,7 @@ class EditorFactory {
         if gutenbergSettings.shouldAutoenableGutenberg(for: post) {
             gutenbergSettings.setGutenbergEnabled(true, for: post.blog)
             gutenbergSettings.postSettingsToRemote(for: post.blog)
+            gutenbergSettings.didAutoenableGutenberg()
```

Comment:
If I understood the code correctly, this will set the global `Key.appWideEnabled` to true, which means `wasGutenbergEnabledOnce` will always return true for all blogs and `shouldAutoenableGutenberg` will return `false` for any other blog.

Reply:
That's true. This was the behavior we agreed on.
We want to show the dialog just one per user.

I kept the per-site flag to avoid showing the dialog when the user manually opted in to gutenberg in that blog, but still show it on the next blog where gutenberg was auto-enabled.

Reply:
> We want to show the dialog just one per user

I can agree with that, but just to double check, this is not just about presenting the dialog but automatically enabling gutenberg when you open a gutenberg post. In previous iterations this was done for all sites when the dialog was shown. Now it only enables it for the current site (which is better), but any other site where the user opens a Gutenberg post will not get gutenberg auto-enabled for new posts. Is this the behaviour we want?